resources/patches/inox-patchset/chromium-clang-r4.patch

```diff
@@ -0,0 +1,11 @@
+--- a/device/fido/u2f_ble_transaction.cc
```

Comment:
So this is a newer version of `ungoogled-chromium/linux/fix-libstdcxx630-errors.patch`? If so, we should probably remove that patch.

Based on this patch's name, I'm given the impression that this is a fix for using clang. I'm thinking that it may be more appropriate to specify it's a fix for libstdc++, because libc++ doesn't have this problem.

Reply:
Huh, I didn't realize both comments appeared. My tab crashed several moments after I posted the other comment, so I refreshed and didn't see it. Hence, this one was created.